### PR TITLE
rename auto batching cli

### PR DESCRIPTION
**Pull request by MarinPostma**
rename `--enable-autobatching` to `--enable-auto-batching`.

as per https://github.com/meilisearch/specifications/pull/96#issuecomment-1060693721

### CODE REVIEW

**Review by gmourier — APPROVED** [review]
Thanks @MarinPostma 🏎 !!!!

**Review by irevoire — APPROVED** [review]
:+1:

**Review by irevoire — APPROVED** [review]

**Comment by MarinPostma** [comment]
Bors merge

File: meilisearch-lib/src/options.rs [hunk 1]
```diff
@@ -48,7 +48,7 @@ pub struct IndexerOpts {
 pub struct SchedulerConfig {
     /// enable the autobatching experimental feature
     #[clap(long, hide = true)]
-    pub enable_autobatching: bool,
+    pub enable_auto_batching: bool,
 
     // The maximum number of updates of the same type that can be batched together.
     // If unspecified, this is unlimited. A value of 0 is interpreted as 1.
```
Comment (Kerollmops) [inline meilisearch-lib/src/options.rs:54]: BTW these are not documentation comment but simple comments.

**Comment by curquiza** [comment]
bors merge

**Pull request merged**